website/views/chatbot_api.py

```
@@ -0,0 +1,148 @@
+            "They help maintain transparency and motivation for contributors."
+        )
+
+    if msg.startswith("/bid") or "bid" in msg:
+        return (
+            "The BLT bidding system lets contributors place bids on issues. "
+            "Maintainers can choose contributors based on bid context."
+        )
+
+    if msg.startswith("/bacon") or "bacon" in msg:
+        return (
+            "Bacon Tokens are BLT’s internal reward currency for contributing to issues, "
```

Ruff (0.14.7)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)